### ISSUE TITLE
feat: tooltip on hover

### ISSUE THREAD

**Issue by a0v0** [zsviczian/excalibrain#129]
![image](https://github.com/zsviczian/excalibrain/assets/62926341/690e25ae-b79f-4d80-b799-3cf64647db9e)

**Comment by zsviczian** [comment]
I agree, this would be nice to have, however, as with the animation feature, I see this as a low priority. You can now get a hover preview of the page and you may add a summary section at the top of the page so it is shown first in the hover preview.

<img width="757" alt="image" src="https://github.com/zsviczian/excalibrain/assets/14358394/fb138c68-3ab5-4aff-af77-ac71428a3c1d">
